.github/workflows/test-javascript.yml

```
@@ -30,7 +30,7 @@ jobs:
         if: ${{ matrix.os != 'windows-latest' }}
         with:
           # See https://github.com/tree-sitter/tree-sitter/blob/master/cli/emscripten-version
```

@kieran-ryan Same here:
```suggestion
          # See https://github.com/tree-sitter/tree-sitter/blob/v0.24.6/cli/loader/emscripten-version
          # (choose the Git tag that corresponds to the version of web-tree-sitter specified in package.json) 
          # Should be kept in sync with .github/workflows/release-npm.yml
```